src/core/MOM.F90

```diff
@@ -5,7 +5,7 @@ module MOM
 
 ! Infrastructure modules
 use MOM_array_transform,      only : rotate_array, rotate_vector
-use MOM_debugging,            only : MOM_debugging_init, hchksum, uvchksum
+use MOM_debugging,            only : MOM_debugging_init, hchksum, uvchksum, totalTandS
```

Comment:
Since the calls to `totalTandS` are in comments, perhaps it also makes sense for the import to be in a comment?

Reply:
I think that this is benign either way.

Reply:
This could come back if we turn on `-Wall` (warning of unused modules loads), but we can deal with that down the road.